pybleau/app/model/dataframe_plot_manager.py

```diff
@@ -548,6 +604,36 @@ def action_delete_requested(self, manager, attr_name, new):
         desc = self._get_desc_for_menu_manager(manager)
         desc.container_idx = CONTAINER_IDX_REMOVAL
 
+    @on_trait_change("contained_plots:plot_factory:context_menu_manager:"
+                     "template_requested", post_init=True)
+    def action_template_requested(self, manager, attr_name, new):
+        """ A factory requested a plot template be created. Launch dialog.
+        """
+        interactor = self.template_interactor
+        if interactor is None:
+            return
+
+        desc = self._get_desc_for_menu_manager(manager)
+        if desc is None:
+            return
+
+        template_name = self._request_template_name_with_desc(desc)
+        if template_name is None:
+            return
+
+        filepath = os.path.join(interactor.get_template_dir(), template_name +
+                                interactor.get_template_ext())
+        saver = interactor.get_template_saver()
+        saver(filepath=filepath, object_to_save=desc.plot_config)
```

Comment:
It may be necessary to specify the arguments for the result from the `get_template_saver` implementation. I am currently unsure how to do that, though.

> Also, because we don't control the `saver` definition, a try and except around it, logging information about failure would be useful.